### DISCUSSION TITLE
Test Base Account Billing

### DISCUSSION THREAD

**Discussion by lordarokh** [microsoft/testbase#37] [Announcements]
Hi There,

Does anybody know, how the Test Base Service billing works?
What are we getting charged for?

- Does number of test packages that have been created (but not tested), have impact on the billing?
- Does the amount of storage that is used to create packages have impact on billing?
- Does the amount of APIs calls have impact on billing? 
- Does number of OS versions that has been configured (but NOT tested) on the package have impact on billing?
- Does the "Verification" process when a package is created, have any impact on billing?
- Or Only for the amount of time that lapsed to perform a certain package testing?

We are currently working for on an internal automation tool for Microsoft Azure Test Base. 
Last 10 days, we have created ( only creation ) multiple different Test Base packages, however we did NOT perform any tests with these packages.
And yet we saw our billing went up 8k/10k during this period.

Thanks in advance.

**Comment by Tinacyt** [comment]
Hi there,

Thank you for the questions! You can find our billing plan at [https://www.microsoft.com/en-us/testbase/pricing](url).

For more detailed Test Base Billing information on your case, I would recommend you to reach out to [testbase-billing@microsoft.com](mailto:testbase-billing@microsoft.com) for help! The Test Base billing product manager will support you directly with a more in-depth investigation of your actual billing usage. 😊

Regards,
Tina
